### PR TITLE
[Learn] Land the learn page

### PR DESCRIPTION
**Pull request by AndrewJakubowicz**
Fixes: https://github.com/lit/lit.dev/issues/822

### Context

"Land" the **Learn** catalog page. This PR does not remove the tutorials page, as that page is still valuable for the tutorials collection.

### Changes

 - Change nav bar `Tutorials` -> `Learn`.
 - Change links to `/tutorials` -> `/learn#filter=tutorial`.

### Test plan

Added a smoke test integration test, which renders the tutorial catalog.

### CODE REVIEW

File: packages/lit-dev-server/src/redirects.ts [hunk 1]
```diff
@@ -35,8 +35,8 @@ export const pageRedirects = new Map([
   // Old release-notes redirects to new v2 location.
   ['/docs/releases/release-notes/1.3.0/',      '/docs/v2/releases/release-notes/1.3.0/'],
   ['/docs/releases/release-notes/1.2.0/',      '/docs/v2/releases/release-notes/1.2.0/'],
-  // Old Lit tutorial redirect to new tutorial catalog
-  ['/tutorial/', '/tutorials/'],
+  // Old Lit tutorial redirect to new learn catalog
+  ['/tutorial/', '/learn/'],
```
Comment (augustjk) [inline packages/lit-dev-server/src/redirects.ts:39]: should there also be one that redirects the plural to /learn?
```suggestion
  ['/tutorial/', '/learn/'],
  ['/tutorials/', '/learn/'],
```

**Review by augustjk — APPROVED** [review]
🧑‍🏫 🚀

File: packages/lit-dev-server/src/redirects.ts [hunk 2]
```diff
@@ -97,12 +97,12 @@ export const oldLitElementSiteRedirects = new Map([
   ['/guide/build',      '/docs/v1/tools/build/'],
   ['/guide/community',  '/docs/v1/resources/community/'],
 
-  ['/try',              '/tutorials/'],
-  ['/try/create',       '/tutorials/'],
-  ['/try/properties',   '/tutorials/'],
-  ['/try/logic',        '/tutorials/'],
-  ['/try/events',       '/tutorials/'],
-  ['/try/style',        '/tutorials/'],
+  ['/try',              '/learn/'],
```
Comment (justinfagnani) [inline packages/lit-dev-server/src/redirects.ts:101]: `/try/` was the old playground/tutorial page. I think it should link to `learn/#filter=tutorial`

File: packages/lit-dev-server/src/redirects.ts (hunk 1 above)
Comment (justinfagnani) [inline packages/lit-dev-server/src/redirects.ts:39]: Shouldn't this be `learn/#filter=tutorial`?

**Review by justinfagnani — APPROVED** [review]

File: packages/lit-dev-server/src/redirects.ts (hunk 2 above)
Comment (AndrewJakubowicz) [reply packages/lit-dev-server/src/redirects.ts]: Thank you! Done

File: packages/lit-dev-server/src/redirects.ts (hunk 1 above)
Comment (AndrewJakubowicz) [reply packages/lit-dev-server/src/redirects.ts]: Done!

**Comment by augustjk** [comment]
Now that the article page is navigable, check links is failing on the hidden links to tags.
Currently tag links are part of the page but hidden with css: https://github.com/lit/lit.dev/blob/397df96ac5e1e2439955125308064f893bf94cde/packages/lit-dev-content/site/css/articles.css#L204

We should just not render them at all until it's available by commenting out: https://github.com/lit/lit.dev/blob/397df96ac5e1e2439955125308064f893bf94cde/packages/lit-dev-content/site/_includes/article.html#L10-L16

**Comment by augustjk** [comment]
Oh no.. the link checker thinks `#` are only meant to be sections on the page, not data to be read with JS.
```
BROKEN REDIRECT /learn/#filter=tutorial
Could not find section matching hash #filter=tutorial.
Searched in file ../lit-dev-content/_site/learn/index.html
```
We may need to explicitly bypass this here
https://github.com/lit/lit.dev/blob/f3febcf7d1ada0d15154c77a0448003d106adbfd/packages/lit-dev-tests/src/check-redirects.ts#L83-L91

**Comment by AndrewJakubowicz** [comment]
@augustjk Thank you for the redirect tip. I've added a small workaround so that we pass the tests.

**Review by augustjk — APPROVED** [review]
Let's land it!! Hype!

**Pull request merged**